### PR TITLE
Staff View: Dissolution Filing SP/GP#11411 | comments

### PR DESCRIPTION
**Pull request by shabeeb-aot**
*Issue #:* /bcgov/entity#11411

*Description of changes:*
* Added staff comments
* Redirection for SP/GP filing from App.vue

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/common/EntityInfo.vue
Comment (severinbeauvais) [inline src/components/common/EntityInfo.vue:129]: I think you don't need this. The template can see `getBusinessId` I think.

File: src/components/common/EntityInfo.vue
Comment (severinbeauvais) [inline src/components/common/EntityInfo.vue:154]: Does the alignment of the staff comment component work ok when you change the screen width?

Comment (severinbeauvais) [reply src/components/common/EntityInfo.vue:154]: I recommend (but don't insist on) just adding class `ml-n3` to the template... IMO, layout changes like this are easier to manage if they are in the template.

File: src/mixins/index.ts
Comment (severinbeauvais) [inline src/mixins/index.ts:2]: If this works then I just learned something :) Thanks!

File: src/views/DissolutionFirm/DissolutionFirm.vue
Comment (severinbeauvais) [inline src/views/DissolutionFirm/DissolutionFirm.vue:51]: You don't need the fallback value since startDateRules is a getter and always has a value.

File: src/views/DissolutionFirm/DissolutionFirm.vue
Comment (severinbeauvais) [inline src/views/DissolutionFirm/DissolutionFirm.vue:336]: This is correct code, but if there will be no other code here, you can just do the assignment in the template, eg `@emitDateSync="dissolutionDate = $event"`.

I leave this up to you.

**Review by severinbeauvais — APPROVED** [review]
I identified some possible cleanup (up to author's discretion), otherwise LGTM.

I note that this is a partial feature implementation since you are not yet doing anything with the dissolution date value.

**Comment by severinbeauvais** [comment]
Also, you need to fix the 2 failing checks before this is merged,

**Review by cameron-freshworks — APPROVED** [review]

**Pull request merged**
